### PR TITLE
Added methods for delete congratulations

### PR DESCRIPTION
**Pull request by RomanKlishch**
- added method for delete congratulation by id of card;
- added method for delete congratulation by id of congratulation;
- added tests;
- removed class PageGenerator
- removed old test for servlet

### CODE REVIEW

File: src/test/java/com/greetingcard/web/servlet/congratulation/AddCongratulationServletTest.java [hunk 1]
```diff
@@ -49,45 +49,4 @@
     @InjectMocks
     private AddCongratulationServlet addCongratulationServlet;
```
Comment (Yuliia-Sh) [inline src/test/java/com/greetingcard/web/servlet/congratulation/AddCongratulationServletTest.java:51]: why tests were deleted?

File: src/test/java/com/greetingcard/web/servlet/user/EditProfileUserServletTest.java [hunk 2]
```diff
@@ -35,32 +35,4 @@
     @Mock
     private HttpSession session;
```
Comment (Yuliia-Sh) [inline src/test/java/com/greetingcard/web/servlet/user/EditProfileUserServletTest.java:37]: why were tests deleted?

File: src/test/java/com/greetingcard/web/servlet/user/LoginServletTest.java [hunk 3]
```diff
@@ -39,55 +39,6 @@
     @Mock
     private User user;
```
Comment (Yuliia-Sh) [inline src/test/java/com/greetingcard/web/servlet/user/LoginServletTest.java:41]: why were test deleted?

File: src/main/resources/application.properties [hunk 4]
```diff
@@ -18,8 +18,6 @@ log.path=logs
...
 iteration=1
 
-pathToFiles="/"
-
 #Path to files saving
-img.storage.path=src/main/webapp/static/img/
-audio.storage.path=src/main/webapp/static/audio/
\ No newline at end of file
+img.storage.path=src/main/webapp/static/
+audio.storage.path=src/main/webapp/static/
```
[the remark this answers is not in the document]
Comment (AlexKievDeveloper) [reply src/main/resources/application.properties]: Please use previous paths or empty paths values:
#Path to files saving
img.storage.path=src/main/webapp/static/img/
audio.storage.path=src/main/webapp/static/audio/

File: src/test/java/com/greetingcard/dao/jdbc/JdbcCongratulationDaoITest.java [hunk 5]
```diff
@@ -32,8 +35,11 @@ void init() {
     }
 
     @AfterEach
-    void afterAll() {
+    void afterAll() throws IOException {
         flyway.clean();
+        Files.deleteIfExists(Path.of("src/main/webapp/static/audio"));
+        Files.deleteIfExists(Path.of("src/main/webapp/static/picture"));
+        Files.deleteIfExists(Path.of("src/main/webapp/static"));
```
Comment (AlexKievDeveloper) [inline src/test/java/com/greetingcard/dao/jdbc/JdbcCongratulationDaoITest.java:42]: Please use previous path 
img.storage.path=src/main/webapp/static/img/
audio.storage.path=src/main/webapp/static/audio/

File: src/main/java/com/greetingcard/dao/jdbc/JdbcCongratulationDao.java [hunk 6]
```diff
@@ -154,4 +137,40 @@ void saveLinks(List<Link> linkList, Connection connection) throws SQLException {
...
+                        String pathToFiles;
+                        String file = resultSet.getString("link");
+                        LinkType linkType = LinkType.getByNumber(resultSet.getInt("type_id"));
+                        if (linkType == LinkType.PICTURE) {
+                            pathToFiles = pathToPictureFiles;
+                        } else {
+                            pathToFiles = pathToAudioFiles;
+                        }
+                        try {
+                            Files.deleteIfExists(Paths.get(pathToFiles, file));
```
Comment (AlexKievDeveloper) [inline src/main/java/com/greetingcard/dao/jdbc/JdbcCongratulationDao.java]: In DB you can use path from Link.getLiink()

**Review by AlexKievDeveloper — APPROVED** [review]

**Review by AlexKievDeveloper — APPROVED** [review]

**Review by Yuliia-Sh — APPROVED** [review]

**Review by OleksandrGubariev — APPROVED** [review]

**Pull request merged**